examples/samples/sessionState.ts

```diff
@@ -0,0 +1,146 @@
+
+  We take for example the context of an online shopping app and see how we can use Session State
+  to implement the maintaining of shopping cart information completely on server side i.e.,
+  remembering the user's shopped items even if they leave the site and return later.
+
+  The scenario in sample walks through user activity of two customers Alice and Bob.
+  Alice shops for 3 items and checks out, whereas Bob adds 3 items and leaves without
+  checking out to likely return later.
+  The session state keeps track of the cart items accordingly.
+*/
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(connectionString);
+
+  await runScenario();
+
+  await ns.close();
+  console.log("\n>>>> sample Done!!!!");
+}
+
+async function runScenario(): Promise<void> {
+  // User activity data for Alice and Bob
+  const shoppingEventsDataAlice = [
+    { event_name: "Add Item", event_details: "Milk" },
+    { event_name: "Add Item", event_details: "Bread" },
+    { event_name: "Add Item", event_details: "Eggs" },
+    { event_name: "Checkout", event_details: "Success" }
+  ];
+
+  const shoppingEventsDataBob = [
+    { event_name: "Add Item", event_details: "Pencil" },
+    { event_name: "Add Item", event_details: "Paper" },
+    { event_name: "Add Item", event_details: "Stapler" }
+  ];
+
+  // Simulating user events
+  await sendMessagesForSession(shoppingEventsDataAlice, "alice");
+  await sendMessagesForSession(shoppingEventsDataBob, "bob");
+
+  await processMessageFromSession("alice");
+  await processMessageFromSession("alice");
+
+  // Displaying snapshot of Alice's shopping cart (SessionState) after processing 2 events
+  // This will show two items
+  await getSessionState("alice");
+
+  await processMessageFromSession("alice");
+  await processMessageFromSession("alice");
+  await processMessageFromSession("bob");
+  await processMessageFromSession("bob");
+  await processMessageFromSession("bob");
+
+  // Displaying snapshot of Alice's shopping cart (SessionState) after processing remaining events
+  // This will show null as Alice checksout and cart is cleared
+  await getSessionState("alice");
+  // Displaying snapshot of Bob's shopping cart (SessionState) after processing all the events
+  // This will show three items
+  await getSessionState("bob");
+}
+
+async function getSessionState(sessionId: string): Promise<void> {
+  const client = ns.createQueueClient(userEventsQueue, { receiveMode: ReceiveMode.peekLock });
+  const messageSession = await client.acceptSession({ sessionId: sessionId });
+
+  const sessionState = await messageSession.getState();
+  // Get list of items
+  console.log(`\nItems in cart for ${sessionId}: ${sessionState}\n`);
+
+  await messageSession.close();
+  await client.close();
+}
+
+async function sendMessagesForSession(shoppingEvents: any[], sessionId: string): Promise<void> {
+  const client = ns.createQueueClient(userEventsQueue, {
+    receiveMode: ReceiveMode.peekLock
```

Comment:
This is the default mode, we can skip passing the `options` altogether. This applies to the other place where we use queueClient as well